plugins/kubernetes/src/components/KubernetesContent/KubernetesContent.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { Typography, Grid } from '@material-ui/core';
+import {
+  InfoCard,
+  Page,
+  pageTheme,
+  Content,
+  ContentHeader,
+  useApi,
+} from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+import { kubernetesApiRef } from '../../api/types';
+
+// TODO this is a temporary component used to construct the Kubernetes plugin boilerplate
+
+export const KubernetesContent: FC<{ entity: Entity }> = ({ entity }) => {
+  const kubernetesApi = useApi(kubernetesApiRef);
+  const [kubernetesObjects, setKubernetesObjects] = useState<{} | undefined>(
+    undefined,
+  );
+  const [error, setError] = useState<string | undefined>(undefined);
+
+  useEffect(() => {
+    kubernetesApi
+      .getObjectsByServiceId(entity.metadata.name)
```

Comment:
so this assumes that the component entity has a name that is also bound in k8s somehow right - what about the namespace?

Reply:
We will probably do what the spotify backstage does and just search in all namespaces and just label selectors to find what we are looking for. 

We can probably add another annotation to limit to the search to a certain namespace in the future for performance but it should probably be optional, as otherwise we're kind of mandating a component is always only in one namespace which might not be true for everyone.

Reply:
OK, so this is related to the other comment thread. What you say with label selectors is interesting then, because then we need a place to put those as well.